main/src/main/java/org/mobilitydata/gtfsvalidator/validator/TripAndShapeDistanceValidator.java

```diff
@@ -139,7 +139,8 @@ static class TripDistanceExceedsShapeDistanceNotice extends ValidationNotice {
   }
 
   /**
-   * The distance between the last shape point and last stop point is less than the 11.1m threshold.
+   * The distance between the last shape point and last stop point is greater than 0 but less than
+   * the 11.1m threshold.
```

Comment:
Out of scope, and I guess I should know that by now, but why do we sometimes talk about 11.1m and sometimes 1.11m?

Reply:
it's a long story :) We defined different thresholds per notice. Specific references to the rationale for each are in the 5.0 release notes: https://github.com/MobilityData/gtfs-validator/releases/tag/v5.0.0